editor/src/components/shared/project-components.ts

```diff
@@ -77,6 +78,10 @@ export function clearInsertableComponentUniqueIDs(
   }
 }
 
+export interface InsertableComponentSamples {
```

Comment:
shall this be renamed to `InsertableComponentGroupSamples` to be consistent with the other siblings?

Reply:
great idea! renamed it in https://github.com/concrete-utopia/utopia/pull/3550/commits/7021566fc9ef1b8c7c538b2b61eeb1a051811451